includes/class-amp-story-post-type.php

```diff
@@ -1384,6 +1386,30 @@ function( $rule ) {
 		}
 	}
 
+	/**
+	 * Add RSS feed link for stories, if theme supports automatic-feed-links.
+	 *
+	 * @since 1.2
+	 */
+	public static function print_feed_link() {
+		if ( ! current_theme_supports( 'automatic-feed-links' ) ) {
```

Comment:
This condition is being used in core's `feed_links()` function, but I'm not sure it makes sense here.

Thinking about it more, I'm thinking that the theme support was added to prevent double-printing the feed links for themes that used to manually add it to the `head`, similar to the story for the `title-tag` theme support.

So this condition should probably be removed. Agreed?